ignite/services/plugin/template/go.mod.plush

```diff
@@ -7,4 +7,7 @@ require (
 	github.com/ignite/cli v0.25.2-0.20221031170327-2d623a4e00ae
 )
 
-replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+replace (
+	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+	github.com/ignite/cli => github.com/joshLong145/cli v0.25.2-0.20221028151152-3c4960fb80e1
```

Comment:
We will need to change his replace back

Reply:
I was asked to add this replace and to update after it was merged

Reply:
thanks to that change we can test the feature locally.

Reply:
Unfortunately, the hash has changed with the rebase/new PR, please use `v0.25.2-0.20221101190607-03fa04134366` now.